src/client/modules/counter/containers/Counter.jsx

```diff
@@ -9,6 +8,8 @@ import CounterView from '../components/CounterView';
 import COUNTER_QUERY from '../graphql/CounterQuery.graphql';
 import ADD_COUNTER from '../graphql/AddCounter.graphql';
 import COUNTER_SUBSCRIPTION from '../graphql/CounterSubscription.graphql';
+import COUNTER_QUERY_CLIENT from '../graphql/CounterQuery_client.graphql';
```

Comment:
Why the convention change? I thought we have started to use `.client.graphql` in the projects, not `_client.graphql`, why this change?